vue/src/components/Uploader/Uploader.vue

```diff
@@ -455,107 +533,36 @@
             this.retries = 0;
             if (response.data.errors && response.data.errors.length) {
               response.data.errors.forEach((error) => {
-                console.log("error", error.message || error);
+                console.error("error", error.message || error);
                 // if there was an error it's probably because the upload isn't finished yet
-                // mark the metadata as dirty so it will save metadata again
-                this.retries++;
+                // mark the metadata as dirty, so it will save metadata again
+                this.retries += 1;
                 this.isDirty = true;
               });
             }
           })
           .catch((error) => {
             this.showError("Error saving metadata");
-            console.log("error", error);
+            console.error("error", error);
           });
       },
       getCSRFToken() {
         const name = "csrftoken";
         if (document.cookie && document.cookie !== "") {
           const cookies = document.cookie.split(";");
-          for (let i = 0; i < cookies.length; i++) {
+          for (let i = 0; i < cookies.length; i += 1) {
             const cookie = cookies[i].trim();
-            if (cookie.substring(0, name.length + 1) === name + "=") {
+            if (cookie.substring(0, name.length + 1) === `${ name }=`) {
               return decodeURIComponent(cookie.substring(name.length + 1));
             }
           }
         }
         return null;
       },
       formatHelpText(title, body, signature) {
-        let text = "<b>" + title + "</b><br><br>" + body;
-        if (signature) {
-          text += "<br><br>" + signature;
-        }
-        return text;
+        return `${title}.\n${body}${signature ? `\n${signature}`:''}`;
```

Comment:
@molund This is a breaking change. Uiv (component library) was updated to the version compatible with Vue3 and its tooltip now escapes html tags in its text property. //n doesn't work as well. 
This breaks the formatting for tooltips, and I'm not sure what the best way to fix it would be. Please let me know if you have any ideas about it.

Reply:
Good point, we can try using [popover](https://uiv.wxsm.space/components/popover.html) component for it. I suggest that we log a separate bug ticket for it.